lib/helpers/ScriptUiHelper.ts

```diff
@@ -0,0 +1,83 @@
+﻿import {Page, Locator} from "@playwright/test"
+
+export class ScriptUiHelper {
+  private readonly page: Page;
+  private readonly newEmptyScriptBtn: Locator;
+  private readonly insertScriptName: Locator;
+  private readonly saveBtn: Locator;
+  private readonly deleteFolderBtn: Locator;
+  private readonly caretBtn: Locator;
+  private readonly scriptTextArea: Locator;
+  private readonly deleteBtn: Locator;
+  private readonly confirmToDeleteBtn: Locator;
+  private readonly insertFolderName: Locator;
+  private readonly createFolderBtn: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.newEmptyScriptBtn = page.getByLabel('New empty script');
+    this.insertScriptName = page.getByLabel('template name');
+    this.saveBtn = page.getByLabel('Save');
+    this.caretBtn = page.locator('div').filter({hasText: 'Scripts'}).locator('#caret-button');
+    this.scriptTextArea = page.locator('textarea.inputarea')
+    this.deleteBtn = page.getByRole('button', {name: 'Delete'});
+    this.confirmToDeleteBtn = page.locator('#confirm').getByLabel('Delete');
+    this.createFolderBtn = page.getByLabel('Create folder...', {exact: true});
+    this.insertFolderName = page.locator('[headline="Create Folder"] >> input');
+    this.deleteFolderBtn = page.getByLabel('Delete');
+  }
+
+  async openActionsMenuForName(name: string) {
```

Comment:
I think this should be named `openActionsMenuForScript` instead of `...ForName` since `ForName` is a bit vague 😄 

The same goes for `clickCaretButtonForName`